spec/controllers/students_controller_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.describe StudentsController, type: :controller do
+    #save instance of student in test db 
```

Comment:
comments like this are not really helpfull .. can remove it.
If you want to pass something to the PR review .. as it as a PR comment inline where the code